tests/upload_files.rs

```diff
@@ -1,33 +1,17 @@
 mod fixtures;
 
-use assert_cmd::prelude::*;
-use assert_fs::fixture::TempDir;
-use fixtures::{port, tmpdir, Error};
+use fixtures::{server, Error, TestServer};
 use reqwest::blocking::{multipart, Client};
 use rstest::rstest;
 use select::document::Document;
 use select::predicate::{Attr, Text};
-use std::process::{Command, Stdio};
-use std::thread::sleep;
-use std::time::Duration;
 
-#[rstest]
-fn uploading_files_works(tmpdir: TempDir, port: u16) -> Result<(), Error> {
+#[rstest(server(&["-u"]))]
```

Comment:
Isn't this the [old compact syntax](https://docs.rs/rstest/0.11.0/rstest/attr.rstest.html#old-compact-syntax)? If so, could you adopt this to the new way? I suppose what you're doing here is partial injection and the new syntax for that is [here](https://docs.rs/rstest/0.11.0/rstest/attr.fixture.html#partial-injection).

If you agree, please change this here and in other places where you use this idiom.